client/tlsutil/tlsconfig.go

```diff
@@ -47,21 +47,12 @@ import (
 type TLSInfo struct {
 	CertFile           string
 	KeyFile            string
-	CAFile             string // TODO: deprecate this in v4
```

Comment:
Does it work if there is no CAfile? like pdctl no specified the `--cacert`.

Reply:
https://github.com/tikv/pd/pull/8347/files#diff-91dc9b32b58d11d334b55b1e6fdc4495c5d28d42f125320b8611d43238799bbcR160 still works. It will be assigned to TrustedCAFile.